src/coredump/coredumpctl.c

```diff
@@ -1096,6 +1111,9 @@ static int run(int argc, char *argv[]) {
                        ansi_highlight_red(),
                        units_active, units_active == 1 ? "unit is running" : "units are running",
                        ansi_normal());
+
+        strv_free(arg_file);
```

Comment:
please add STATIC_DESTRUCTOR_REGISTER(arg_file, strv_freep); to the top of this file, it's how we do these things these days...

Reply:
Ah, I see, thanks!